apps/src/templates/manageStudents/CodeReviewGroupsDialog.jsx

```diff
@@ -47,7 +47,7 @@ export default function CodeReviewGroupsDialog({
     setGroups(groups);
   };
 
-  useEffect(() => getInitialGroups(), [isDialogOpen]);
+  useEffect(() => getInitialGroups(), [isDialogOpen, getInitialGroups]);
```

Comment:
Not sure I understand why `isDialogOpen` is needed here (but ok to leave since you're just addressing the warning).

Reply:
Good catch, I tried removing it and there were no complaints.. which is weird because I have seen the linter inform that a dep isn't needed, so not sure why that didn't happen with this